### PR TITLE
rimage: Fix DRAM_BASE address for i.MX8M

### PR DESCRIPTION
**Pull request by dianacretu**
As per xtensa configuration mmap, DRAM_BASE is 0x3b6e8000.

Signed-off-by: Diana Cretu <diana.cretu@nxp.com>

### CODE REVIEW

File: rimage/file_simple.c [hunk 1]
```diff
@@ -43,7 +43,7 @@
 #define IMX8M_IRAM_BASE		0x3b6f8000
 #define IMX8M_IRAM_HOST_OFFSET	0x10000
 #define IMX8M_IRAM_SIZE		0x800
-#define IMX8M_DRAM_BASE		0x3b6f8000
+#define IMX8M_DRAM_BASE		0x3b6e8000
```
Comment (paulstelian97) [inline rimage/file_simple.c]: I think we could get both on-DSP DRAM regions by extending the size to 0x10000 on the next line, but I wonder if we should even do that.

Comment (dbaluta) [reply rimage/file_simple.c]: I don't think we use that area, so we are fine with this change. It is similar with what we do for imx8 and imx8x

Comment (paulstelian97) [reply rimage/file_simple.c]: Fair enough. Boot code may use this area but everything else goes in the SDRAM regions so this is fine.

**Comment by dbaluta** [comment]
Travis CI fails because docker image is not updated to contain the i.MX8 Qemu patches.

Cc: @xiulipan

**Pull request merged**